src/xml/DrawingXML.java

```diff
@@ -1,44 +0,0 @@
-package xml;
-
-import view.Path;
-import model.Drawing;
-
-public class DrawingXML {
-	
-	/************************************
-	 *	Attributes
-	 ************************************/
-	private Drawing img;
-	
-	/************************************
-	 *	Constructor
-	 ************************************/
-	public DrawingXML(Drawing img) {
-		this.img = img;
-	}
-	
-	/************************************
-	 *	Getters and Setters
-	 ************************************/
-	public Drawing getImg() {
-		return img;
-	}
-
-	public void setImg(Drawing img) {
-		this.img = img;
-	}
-	
-	/************************************
-	 *	Methods
-	 ************************************/
-	public String getDrawing(){
```

Comment:
Not sure that we should remove that class. I do agree for Path, but not really for this one, as it gives the xml part for the global image. (Which you put in model, or the presentation package xml should be the only place where we should find xml)